src/crawlee/basic_crawler/_basic_crawler.py

```diff
@@ -524,6 +526,14 @@ async def add_requests(
             wait_for_all_requests_to_be_added_timeout=wait_for_all_requests_to_be_added_timeout,
         )
 
+    async def _use_state(self, key: str, default_value: dict | None = None) -> dict:
+        store = await self.get_key_value_store()
+        return await store.get_auto_saved_value(key, default_value)
+
+    async def _save_persist_state(self) -> None:
```

Comment:
I suggest we rename this method to `_save_crawler_state` and the one on the `KeyValueStore` to `persist_autosaved_values`.